### PR TITLE
Feature/dsnd 1462 add delete endpoint

### PR DESCRIPTION
**Pull request by JAndrewCH**

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java [hunk 1]
```diff
@@ -4,9 +4,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (bcullerton) [inline src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java:7]: Could this be changed back to the individual imports?

File: src/test/java/uk/gov/companieshouse/pscstatementdataapi/service/PscStatementServiceTest.java [hunk 2]
```diff
@@ -13,22 +14,33 @@
...
     PscStatementService pscStatementService;
 
+    private PscStatementDocument document;
+
+    @BeforeEach
+    public void setUp() {
+        document = new PscStatementDocument();
+    }
     @Test
     public void statementReturnedByCompanyNumberAndStatementIdFromRepository() throws JsonProcessingException, ResourceNotFoundException {
```
Comment (bcullerton) [inline src/test/java/uk/gov/companieshouse/pscstatementdataapi/service/PscStatementServiceTest.java:45]: Could this test be refactored to use the new global document?

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java [hunk 3]
```diff
@@ -26,4 +24,19 @@ public ResponseEntity<Statement> searchPscStatements(@PathVariable String compan
         Statement statement = pscStatementService.retrievePscStatementFromDb(company_number, statement_id);
         return new ResponseEntity<>(statement, HttpStatus.OK);
     }
+    /**
+     * Delete psc-statement information for a statement id.
+     *
+     * @param  statementId  the statement id to be deleted
+     * @return return 200 status with empty body
+     */
+    @DeleteMapping("/company/{company_number}/persons-with-significant-control-statement/{statement_id}/Internal")
```
Comment (bcullerton) [inline src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java:33]: Could the internal be changed to lower case here

**Review by bcullerton — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java (hunk 3 above)
Comment (JAndrewCH) [reply src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java:33]: Can do, it was uppercase on the JIra but I can just add a comment

**Review by bcullerton — APPROVED** [review]

**Pull request merged**